index.md

```diff
@@ -22,13 +22,19 @@ visual and accessible presentations; to use the same documents in many
 different applications for different vendors; and to lengthen the time
 in which the documents are likely to be useable.
 
-While documents with declarative markup are often encoded in XML, XML
-vocabularies do not need to be declarative and declarative markup does
-not need to be in XML. The web is full of voices that champion clean,
-readable HTML documents styled with CSS. The academic fields of
-Digital Humanities and Corpus Linguistics, as well as companies from
-publishing houses to aircraft manufacturers, are developing more and
-more innovative uses for declarative markup.
+The time is ripe now to explain declarative markup: for a few years now, 
```

Comment:
Minor edit: I suggest deleting both instances of the word "now" - they're superfluous.

Reply:
I incorporated your suggestions with one change: not all enterprises need open data policies. so tweaked your sentence. please take a look. (and also deleted the 2 "now")